io.qt.QtCreator.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id": "io.qt.QtCreator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "qtcreator",
+    "rename-desktop-file": "org.qt-project.qtcreator.desktop",
+    "rename-appdata-file": "org.qt-project.qtcreator.appdata.xml",
+    "rename-icon": "QtProject-qtcreator",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "clang",
```

Comment:
Anything wrong with clang provided in the Sdk? It's not the latest but not that old either, `10.0.1`.